umh.docs.umh.app/content/en/docs/features/unified-namespace/index.md

```diff
@@ -0,0 +1,60 @@
+{{% /notice %}} 
+
+In the United Manufacturing Hub, each single piece of information / "message" / "event" is sent through a message broker, which is also called the Unified Namespace.
+
+## What can I do with it?
+
+The Unified Namespace / Message Broker in the United Manufacturing Hub provides several notable functionalities in addition to the features already mentioned:
+- Easy integration using MQTT: Many modern shopfloor equipment can send and receive data using the MQTT protocol. 
+- Easy integration with legacy equipment: Using tools like [Node-RED](/docs/architecture/microservices/core/node-red/), data can be easily extracted from various protocols such as Siemens S7, OPC-UA, or Modbus
+- Get notified in real-time via MQTT: The Unified Namespace allows you to receive real-time notifications via MQTT when new messages are published. This can be useful for applications that require near real-time processing of data, such as an AGV waiting for new commands.
+- Retrieve past messages from Kafka logs: By looking into the Kafka logs, you can always be aware of the last messages that have been sent to a topic. This allows you to replay certain scenarios for troubleshooting or testing purposes.
+- Efficiently process messages from millions of devices: The Unified Namespace is designed to handle messages from millions of devices in your factory, even over unreliable connections. By using Kafka, you can efficiently at-least-once process each message, ensuring that no data is lost or duplicated.
+- Trace messages through the system: The Unified Namespace provides tracing capabilities, allowing you to understand where messages are coming from and where they go. This can be useful for debugging and troubleshooting purposes. You can use the [Management Console](https://mgmt.docs.umh.app/docs/) to visualize the flow of messages through the system.to visualize the flow of messages through the system.
+
+## How can I use it?
+
+Using the Unified Namespace is quite simple: 
+
+Configure your IoT devices and devices on the shopfloor to use the in-built MQTT broker of the United Manufacturing Hub by specifying the MQTT protocol, selecting unencrypted (1883) / encrypted (8883) ports depending on your configuration, and send the messages into a topic starting with `ia/raw`. From there on, you can start processing the messages in Node-RED by reading in the messages again via MQTT or Kafka, adjusting the payload or the topic to match the [UMH datamodel](/docs/architecture/datamodel/) and sending it back again to MQTT or Kafka.
+
+If you send the messages into other topics, some features might not work correctly (see also [limitations](#what-are-the-limitations)).
+
+{{% notice tip %}}
+**Recommendation:** Send messages from IoT devices via MQTT and then work in Kafka only.
+{{% /notice %}} 
+
+## What are the limitations?
```

Comment:
Max 249 chars (https://github.com/apache/kafka/commit/ad3dfc6ab25c3f80d2425e24e72ae732b850dc60)
Legal char regex: [a-zA-Z0-9\\._\\-]

Reply:
Max message size defaults to 1000000 byte (https://docs.confluent.io/4.0.0/clients/librdkafka/CONFIGURATION_8md.html)